components/availability/src/Availability.svelte

```diff
@@ -331,19 +337,22 @@
         let availability = AvailabilityStatus.FREE; // default
         if (allCalendars.length) {
           for (const calendar of allCalendars) {
-            let availabilityExistsInSlot = calendar.timeslots.some(
-              (block) => time < block.end_time && block.start_time < endTime,
-            );
+            const slot = {
+              start_time: time,
+              end_time: endTime,
+              available_calendars: [],
+            };
+            const slotAvailability = overlap([...calendar.timeslots], slot);
             if (calendar.availability === AvailabilityStatus.BUSY) {
-              if (!availabilityExistsInSlot) {
+              if (slotAvailability.num < capacity) {
```

Comment:
busy calendars must check that the number of events at that time does not exceed capacity

Reply:
Suggestion: make it `<= capacity` -- to me it says "this is the most events I can have at this time", rather than "this is one more than the most events I can have at this time"

Reply:
Nvm -- I was misunderstanding what .num did!

Could you perhaps change .num to `concurrentEvents` or something ?

Reply:
yes!